stdlib/Dates/src/conversions.jl

```diff
@@ -38,26 +38,33 @@ Base.convert(::Type{Day},dt::Date) = Day(value(dt))            # Converts Date t
 
 ### External Conversions
 const UNIXEPOCH = value(DateTime(1970)) #Rata Die milliseconds for 1970-01-01T00:00:00
+localepoch() = value(DateTime(Libc.TmStruct(0))) #Rata Die for local time at UTC 1970-01-01T00:00:00
 
 """
-    unix2datetime(x) -> DateTime
+    unix2datetime(x::Real; local_time::Bool=false) -> DateTime
```

Comment:
Don't forget to update the spelling of `localtime` in the docstrings.

Reply:
done